python-frontend/src/test/java/org/sonar/python/semantic/v2/TypeInferenceV2Test.java

```diff
@@ -57,6 +60,63 @@ void test() {
     System.out.println("hello");
   }
 
+  @Test
+  void testTypeshedImports() {
```

Comment:
_Nitpick_ to make our tests more robust I think it would be better to avoid relying too much on statement indexes when parsing a different file.
I think we can also use `PythonTestUtils#parse` or even `parseWithoutSymbols` with code provided as a text block to create the `FileInput` and visit it rather than needing an actual file for the simpler cases.